test/jest-config/jest.config-base-config.js

```diff
@@ -0,0 +1,27 @@
+const baseConfig =  {
+      rootDir: '../..',
+      roots: ['<rootDir>/test'],
```

Comment:
As this is only true for 2 tests and overrided by the rest, I'd probably remove this from the base